vm/chef/cookbooks/joomla/files/joomla

```diff
@@ -14,11 +14,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-source /opt/c2d/c2d-utils || exit 1
+source /opt/c2d/joomla-utils || exit 1
 
-joomla_password="$(get_attribute_value "joomla-db-password")"
-declare -r joomla_password
+declare -r JOOMLA_DB_PASSWORD="$(get_attribute_value "joomla-db-password")"
```

Comment:
not exported vars should be named as lower case

Reply:
done

Reply:
done